nemo_aligner/package_info.py

```diff
@@ -14,9 +14,9 @@
 
 
 MAJOR = 0
-MINOR = 1
+MINOR = 2
 PATCH = 0
-PRE_RELEASE = "rc0"
+PRE_RELEASE = ""
```

Comment:
Should we use something like `0.3.0.dev` since technically we are now working on the next version in `main`?

Reply:
yup! fixed https://github.com/NVIDIA/NeMo-Aligner/commit/62005260051edb5d9ea4794fa1fc3b170155e239